src/data/markdown/docs/07 extensions/03 Guides/build-k6-using-docker.md

```diff
@@ -8,40 +8,40 @@ Using the [xk6 Docker image](https://hub.docker.com/r/grafana/xk6/) can simplify
 
 ## Building your first extension
 
-For example, to build a k6 v0.43.1 binary with the [`xk6-kafka`](https://github.com/mostafa/xk6-kafka) and [`xk6-output-influxdb`](https://github.com/grafana/xk6-output-influxdb) extensions, run one of the commands below, depending on your operating system:
+For example, to build a k6 latest version binary with the [`xk6-kafka`](https://github.com/mostafa/xk6-kafka) and [`xk6-output-influxdb`](https://github.com/grafana/xk6-output-influxdb) extensions, run one of the commands below, depending on your operating system:
 
 <CodeGroup labels={["Linux", "Mac", "Windows PowerShell", "Windows"]}>
 
 ```bash
-docker run --rm -u "$(id -u):$(id -g)" -v "${PWD}:/xk6" grafana/xk6 build v0.43.1 \
+docker run --rm -u "$(id -u):$(id -g)" -v "${PWD}:/xk6" grafana/xk6 build latest \
```

Comment:
I would argue to drop `latest` here and the explanation above.

This way the k6 will build with the highest version  requested by the extensions. And won't fail if the latest version happens to be breaking any of them, but none of them request it.

Reply:
This was indeed my first attempt, but the command fails if we don't specify `latest`, with the same errors as if it was building with a previous version... On my mac:

```bash
 docker run --rm -e GOOS=darwin -u "$(id -u):$(id -g)" -v "${PWD}:/xk6" \
  grafana/xk6 build \
  --with github.com/mostafa/xk6-kafka@v0.17.0 \
  --with github.com/grafana/xk6-output-influxdb@v0.3.0
```

```shell
go: downloading github.com/chromedp/sysutil v1.0.0
k6 imports
	go.k6.io/k6/cmd imports
	github.com/grafana/xk6-output-prometheus-remote/pkg/remotewrite imports
	go.buf.build/grpc/go/prometheus/prometheus: unrecognized import path "go.buf.build/grpc/go/prometheus/prometheus": https fetch: Get "https://go.buf.build/grpc/go/prometheus/prometheus?go-get=1": dial tcp: lookup go.buf.build on 192.168.65.7:53: no such host
2023/08/07 12:33:51 [INFO] Cleaning up temporary folder: /tmp/buildenv_2023-08-07-1233.1381995159
```

Reply:
Bump the versions of the extensions to one that use v0.45.1 
https://github.com/mostafa/xk6-kafka/releases/tag/v0.19.1 
https://github.com/grafana/xk6-output-influxdb/releases/tag/v0.4.1

I even wonder if we should not just put the version to `latest`

Reply:
I guess it was the same thing, we wanted to showcase how to add a concrete version. Let me bump those then

Reply:
Yeah, I think those explicit versions were there to ensure the command would run. Obviously, the latest issue corrected by v0.45.1 kinda breaks that.

Reply:
Done @mstoykov & @javaducky Let me know if that's better or we still would adjust anything